### PR TITLE
Add user verified table + route

### PR DESCRIPTION
**Pull request by andr-dev**

### CODE REVIEW

**Review by leeminkyu16 — APPROVED** [review]
Let's Gamble, Try Merging

File: android/app/src/main/java/org/uwaterloo/subletr/api/apis/UserApi.kt [hunk 1]
```diff
@@ -483,6 +483,74 @@ class UserApi(basePath: kotlin.String = defaultBasePath, client: OkHttpClient =
...
+     * @return kotlin.Boolean
+     * @throws IllegalStateException If the request is not correctly configured
+     * @throws IOException Rethrows the OkHttp execute method exception
+     * @throws UnsupportedOperationException If the API returns an informational or redirection response
+     * @throws ClientException If the API returns a client error response
+     * @throws ServerException If the API returns a server error response
+     */
+    @Suppress("UNCHECKED_CAST")
+    @Throws(IllegalStateException::class, IOException::class, UnsupportedOperationException::class, ClientException::class, ServerException::class)
+    suspend fun userIsVerified() : kotlin.Boolean = withContext(Dispatchers.IO) {
```
Comment (abhaykshar) [inline android/app/src/main/java/org/uwaterloo/subletr/api/apis/UserApi.kt]: need to pass userid for this function so other users can know if a user is verified or not

Comment (andr-dev) [reply android/app/src/main/java/org/uwaterloo/subletr/api/apis/UserApi.kt]: Yep, just added!

**Review by abhaykshar — APPROVED** [review]
nice!

**Pull request merged**